### DISCUSSION TITLE
Cofiguration for new rlink 2 rlan16

### DISCUSSION THREAD

**Discussion by sdade86** [cedricp/ddt4all#1057] [Q&A]
Hello all gentlemen!
I decided to mount a used r-link2 system, version lanr16 281152288R hw: 3.11.5.4 sw: 7.8.24.136 instead of the original lanr14 on my Talisman Initiale 2016.
Once installed I immediately noticed that I was quite lucky as it turned on without any problems with the anti-theft device.
But now there are problems. the HUD is not showed in rlink parameters and in the steering wheel controls only the button to lower the volume and change the source works.
I have already done several tests with ddt4all but I can't get it to work correctly.
Furthermore, I cannot find the correct xml file as the program always tells me that the control unit software does not match perfectly.
Please let me know if you need any other information and how to get it.
Thank you all!

**Comment by AndyIreland** [comment]
SWRC you need to change to match your steering wheel control buttons

**Reply by sdade86** [reply]
Thanks for your answer! can you tell me where is in ddt4all? I can't find it

**Reply by AndyIreland** [reply]
good point, I cannot find it either if its A-IVI v.09 rlink  - unless I have missed it in settings 

all I can find is this where it can test the SWRC buttons:
![image](https://github.com/cedricp/ddt4all/assets/49172217/7d325ad0-5bc8-464e-a824-dc3663666c5a)

**Comment by AndyIreland** [comment]
this might be the setting here for the SWRC:
![image](https://github.com/cedricp/ddt4all/assets/49172217/531d4e7f-00f4-48dd-b684-1640dd545938)

**Reply by sdade86** [reply]
I have only rvlv 4 and rvlv 5 and nothing change between this two setting

**Comment by KarelSvo** [comment]
RLINK 2 = RadNav
 EASYLINK = A-IVI

**Reply by AndyIreland** [reply]
Thanks @KarelSvo  - couldnt even find SWRC settings in RadNav too

**Comment by sdade86** [comment]
I'm back to update you.
The problem was not the coding of SWRC (I think it can't be configured, if I remember correctly, 5 or 6 wire start from SWRC that go directly to rlink2, Therefore they are not CAN line signals.). Anyway the problem is an hardware problem due to whoever disassembled rlink2 from the old car. Luckily all solved with some smd capacitor and soldering skills.
![IMG_6954](https://github.com/cedricp/ddt4all/assets/141132439/275e5735-bd05-4427-bf96-dc936021a2b5)

**Comment by sdade86** [comment]
Thanks to all for the contribute!

**Comment by johnkar87** [comment]
hi i have 2 problems .

1) when i was in v9 larn14 rlink2, sometimes sound volume was was decreasing when for example sensors parking from car alarms so the volume was staying low and not goes up as it normally 

2)now i am in v8  and Air quality widget not exists

so these problem can be fixed with ddt4all parameters?
